includes/Connector.class.php

```diff
@@ -1,17 +1,25 @@
 <?php
 
 require_once(dirname(__FILE__) . "/exceptions/RequestException.php");
+require_once(dirname(__FILE__) . "/exceptions/TimeoutException.php");
+require_once(dirname(__FILE__) . "/exceptions/ClientException.php");
+require_once(dirname(__FILE__) . "/exceptions/ServerException.php");
 
 /**
  * Class AC_Connector
  */
 class AC_Connector {
 
 	/**
-	 * Default curl timeout
+	 * Default curl transfer timeout

```

Comment:
Good point, I'll make this clearer. I think you swapped your two descriptions around, `CONNECTTIMEOUT` is time spent waiting before a connection is made (if the server is 100% down, this will handle the timeout), `TIMEOUT` is time spent transferring data after the connection is made (if the server is overwhelmed and can't process the request, this will handle the timeout).